### PR TITLE
Updates Button usage to use default button

### PR DESCRIPTION
**Pull request by taysea**
Now that new theme styling is in, we don't need to use custom Box styling to achieve the desired Button appearance/behavior.

### CODE REVIEW

**Comment by britt6612** [comment]
Screen Shot 2020-05-08 at 12 22 27 PM
This contrast seems off should this be white ? It is not reading the dark color

**Comment by britt6612** [comment]
Is footer in figma updated to match button styling in here ?

**Comment by taysea** [comment]
> Is footer in figma updated to match button styling in here ?

It should be no border, are you looking at the deploy or on local? If local, you may be on cached version of grommet

**Comment by taysea** [comment]
> Screen Shot 2020-05-08 at 12 22 27 PM
> 
> This contrast seems off should this be white ? It is not reading the dark color

Good catch -- this should probably be an issue filed on grommet.

**Comment by britt6612** [comment]
> > Screen Shot 2020-05-08 at 12 22 27 PM
> > This contrast seems off should this be white ? It is not reading the dark color
> 
> Also due to cached grommet I think.

ignore I didnt have latest grommet on my local this is fine in the deploy

File: aries-site/src/examples/templates/list-views/ListScreenExample.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ const PageHeaderExample = ({ title }) => (
 
 const AppHeaderExample = () => (
   <Header pad={{ vertical: 'small' }}>
-    <Button>
+    <Button plain>
```
Comment (halocline) [inline aries-site/src/examples/templates/list-views/ListScreenExample.js:170]: For the <Text> instances on lines 180 & 183, I think we need to add color="text" -- right now coming across as text-strong

File: aries-site/src/examples/templates/dashboards/DashboardExample.js [hunk 2]
```diff
@@ -194,7 +194,7 @@ const ScreenContainer = ({ ...rest }) => {
 
 const AppHeaderExample = () => (
   <Header pad={{ vertical: 'small' }}>
-    <Button>
+    <Button plain>
```
Comment (halocline) [inline aries-site/src/examples/templates/dashboards/DashboardExample.js:197]: lines 207 & 210 need color="text"

**Pull request merged**